apps/web/utils/actions/permissions.ts

```diff
@@ -12,7 +12,12 @@ const logger = createScopedLogger("actions/permissions");
 
 export const checkPermissionsAction = actionClient
   .metadata({ name: "checkPermissions" })
-  .action(async ({ ctx: { emailAccountId } }) => {
+  .action(async ({ ctx: { emailAccountId, provider } }) => {
```

Comment:
btw `actionClient` likely needs changes. There's also middleware for GET requests in the middleware.ts file that probably needs adjusting too